samples/wifi/sr_coex/sysbuild/hci_rpmsg/prj.conf

```diff
@@ -0,0 +1,32 @@
+CONFIG_IPC_SERVICE=y
```

Comment:
Missing license headers?

Reply:
These files originate from zephyr, so yes the headers do not apply